modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp

```diff
@@ -0,0 +1,1337 @@
+// Tencent is pleased to support the open source community by making ncnn available.
```

Comment:
You did some adaptation. Therefore, it makes sense to put a dedicated comment in the beginning:

```
// This file is part of OpenCV project.
// It is subject to the license terms in the LICENSE file found in the top-level directory
// of this distribution and at http://opencv.org/license.html.

/*
Winograd-based convolution F(6x6, 3x3). The code has been borrowed from ncnn inference engine and adapted for OpenCV by Zihao Mu.

Please, see the original copyright below
*/